static/js/modules/ajax-download.js

```diff
@@ -1,41 +1,17 @@
-var _ = require('underscore');
-
 (function () {
 
   'use strict';
 
-  var FILENAME_REGEX = /filename\=([^;]*)/;
-
-  var getFilename = function(res) {
-    var disposition = res.headers('Content-Disposition');
-    if (disposition) {
-      var match = disposition.match(FILENAME_REGEX);
-      if (match) {
-        return match[1];
-      }
-    }
-  };
-
-  var getHref = function(res) {
-    var data = res.data;
-    if (!_.isString(data)) {
-      data = JSON.stringify(data);
-    }
-    var mimetype = res.headers('Content-Type');
-    return 'data:' + mimetype + ',' + encodeURIComponent(data);
-  };
-
   /**
-   * Prompts the user to download a file given an $http response
+   * Prompts the user to download a file given a url.
    */
-  exports.download = function(res) {
+  exports.download = function(url) {
     var element = document.createElement('a');
-    element.setAttribute('href', getHref(res));
-    element.setAttribute('download', getFilename(res));
+    element.setAttribute('href', url);
+    element.setAttribute('download', 'failed'); // this filename only shows in case of failed download AFAIK
```

Comment:
The server does set it to something relevant (`audit-201703091526.xml` or whatever). So this only shows in case of fail. 
But I guess there could be a case where the server doesn't set it right, and the download would still go through with that default name...
I'll set it to `'download'` instead then.